.github/workflows/review.yaml

```diff
@@ -75,9 +73,7 @@ jobs:
         run:  pip install -r requirements-dev.txt
 
       - name: Run Tests
-        run: python3 -m tools.test --cov
-        env:
-          PYTHONPATH: "./src:."
+        run: ./tools/test
```

Comment:
Should this still have --cov in order to generate coverage?

Reply:
Oh. *that's* what broke #129/#130

Reply:
```suggestion
        run: ./tools/test --cov
```